atm/src/forcing.F90

```diff
@@ -105,12 +107,21 @@ subroutine forcing_init_fields(self, fields, forcing_date, &
             num_land_fields = num_land_fields + 1
         endif
 
+        ! simple scale_constant
+        call self%core%get(fp, "scale_constant", scale_constant, scale_constant_found)
+
+        call assert( .not. (scaling_found .and. scale_constant_found), &
+                     "Cannot have scaling_fiilename and scale_constant")
+
         ! Get the shape of forcing fields
         filename = filename_for_year(filename_template, forcing_date%getYear())
         ! Initialise a new field object.
         if (scaling_found) then
```

Comment:
logic in this section would need changes of both constant and variable scalings were allowed

Reply:
Yes, there would have to be a call with both arguments supplied.